client/app/bundles/KudosApp/components/KudoSelectMenu.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react'
+import Select from 'react-select'
+
+function constructOptions(user) {
+  const filterValue = `${user[0]} ${user[1]}`
+  const displayValue = `${user[0]} (${user[1]})`
+  return { value: filterValue, label: displayValue, email: user[1] }
```

Comment:
Style tip: you can avoid the magic number-y array accesses here if you use an array destructure in the parameter list:

```js
function constructOptions([ name, email ]) {
  ...
}
```

Additional style tip:  if you assign to `value` and `label` instead of `filterValue` and `displayValue`, you can use the object assignment es6 shorthand:
```js
return { value, label, email }
```